homeassistant/components/blebox/sensor.py

```diff
@@ -5,7 +5,9 @@
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 
 from . import BleBoxEntity, create_blebox_entities
-from .const import BLEBOX_TO_UNIT_MAP
+from ...const import TEMP_CELSIUS
```

Comment:
Please use the full import path here `from homeassistant.const import TEMP_CELSIUS`
Relative paths should only be used within the integration/component.